src/core/text/Text.js

```diff
@@ -246,11 +183,14 @@ Object.defineProperties(Text.prototype, {
 
 /**
  * Renders text and updates it when needed
- *
+ * @param respectDirty {boolean} Whether to abort updating the text if the Text isn't dirty and the function is called.
  * @private
  */
-Text.prototype.updateText = function ()
+Text.prototype.updateText = function (respectDirty)
```

Comment:
It's been a while since the original discussion, but I believe I added that in to preserve backwards compatibility at another dev's request.